Project/Assets/ML-Agents/Examples/FoodCollector/Scripts/FoodCollectorAgent.cs

```diff
@@ -139,16 +129,7 @@ public void MoveAgent(ActionSegment<int> act)
             if (shootCommand)
             {
                 m_Shoot = true;
-                dirToGo *= 0.5f;
```

Comment:
This got lost in the conversion, right?

Reply:
Maybe easier to keep everything in one function and just pass the ActionBuffers, instead of doing it individually.